apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/BuildDisplayItem.tsx

```diff
@@ -328,22 +332,18 @@ function ExcludeButton({ id }: { id: string }) {
       title={
         <Box>
           <Typography>{t`excludeArt.excludeArtifactTip`}</Typography>
-          <Typography>
-            <ColorText color={excluded ? 'error' : 'success'}>
-              {t(excluded ? 'excludeArt.excluded' : 'excludeArt.included')}
-            </ColorText>
-          </Typography>
         </Box>
       }
       placement="top"
       arrow
     >
       <Button
         onClick={toggle}
-        color={excluded ? 'error' : 'success'}
+        color={excluded ? 'secondary' : 'success'}
         size="small"
+        startIcon={excluded ? <CheckBoxOutlineBlankIcon /> : <CheckBoxIcon />}
       >
-        {excluded ? <BlockIcon /> : <ShowChartIcon />}
+        <span>{t`excludeArt.included`}</span>
```

Comment:
prob dont need a span anymore

also the wording should be more like "Include this artifact in optimization"

Reply:
done